docs/getting_started/hopsworksai/guides/databricks_quick_start_azure.rst

```diff
@@ -0,0 +1,190 @@
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-9.png
+    :alt: Check the Hopsworks virtual network
+    :target: `azure-databricks-9.png`_
+    :align: center
+    :figclass: align-center
+
+The peering connection should now be listed as initiated:
+
+.. _azure-databricks-11.png: ../../../_images/azure-databricks-11.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-11.png
+    :alt: Peering connection initiated
+    :target: `azure-databricks-11.png`_
+    :align: center
+    :figclass: align-center
+
+On the Azure portal, go to *Virtual networks* and search for the virtual network used by your
+Hopsworks cluster:
+
+.. _azure-databricks-12.png: ../../../_images/azure-databricks-12.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-12.png
+    :alt: Virtual networks
+    :target: `azure-databricks-12.png`_
+    :align: center
+    :figclass: align-center
+
+Open the network and select *Peerings*:
+
+.. _azure-databricks-13.png: ../../../_images/azure-databricks-13.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-13.png
+    :alt: Select peerings
+    :target: `azure-databricks-13.png`_
+    :align: center
+    :figclass: align-center
+
+Choose to add a peering connection:
+
+.. _azure-databricks-14.png: ../../../_images/azure-databricks-14.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-14.png
+    :alt: Add a peering connection
+    :target: `azure-databricks-14.png`_
+    :align: center
+    :figclass: align-center
+
+Name the peering connection and select *I know my resource ID*. Paste the string copied when creating
+the peering from Databricks Azure. If you haven't copied that string, then manually select the virtual
+network used by Databricks and press *OK* to create the peering:
+
+.. _azure-databricks-15.png: ../../../_images/azure-databricks-15.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-15.png
+    :alt: Configure peering
+    :target: `azure-databricks-15.png`_
+    :align: center
+    :figclass: align-center
+
+The peering should now be *Updating*:
+
+.. _azure-databricks-16.png: ../../../_images/azure-databricks-16.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-16.png
+    :alt: Cloud account settings
+    :target: `azure-databricks-16.png`_
+    :align: center
+    :figclass: align-center
+
+Wait for the peering to show up as *Connected*. You should now be able to reach the Feature Store from
+Databricks:
+
+.. _azure-databricks-17.png: ../../../_images/azure-databricks-17.png
+.. figure:: ../../../imgs/hopsworksai/azure-databricks-17.png
+    :alt: Cloud account settings
+    :target: `azure-databricks-17.png`_
+    :align: center
+    :figclass: align-center
+
+.. include:: ../../../featurestore/integrations/guides/databricks.rst
+  :start-after: .. include-2-start
+  :end-before:  .. include-2-stop
+
+Step 3: Connecting to the Feature Store from Databricks
+-------------------------------------------------------
+
+Step 3.1: Installing the hops SDK
```

Comment:
Is that still relevant or do we need to change it with `hsfs` ?

Reply:
I think we should probably offer both options but yes it should then be updated.